### ISSUE TITLE
pkgdown site

### ISSUE THREAD

**Issue by AdamSpannbauer** [bmewing/requirements#40]
Generate a [`pkgdown`](https://github.com/r-lib/pkgdown) (or alternative) site and host via this repo's github pages

**Comment by bmewing** [comment]
I believe this was implemented in #48

**Comment by bmewing** [comment]
There still nothing in the 'environment' did something not work properly?

**Comment by bmewing** [comment]
I got the keys setup (I think) but we need to wait for a merge into master before anything will actually deploy.

**Comment by AdamSpannbauer** [comment]
Where will it be deployed from? I saw the `gh-pages` branch but its currently empty.  I've never used `pkgdown` so I'm curious to see the site/source.

**Comment by bmewing** [comment]
Travis CI should build and deploy it to that branch

**Comment by AdamSpannbauer** [comment]
We should link to this in the readme once it’s confirmed to be built correctly. Will it be at [bmewing.github.io/requirements](https://bmewing.github.io/requirements)?

**Comment by bmewing** [comment]
It should be once the build is complete.

**Comment by bmewing** [comment]
It is up at the link you included.

**Comment by AdamSpannbauer** [comment]
Looking good. Do you know if there’s an automatic way to include all vignettes in the site automagically?

**Comment by bmewing** [comment]
The one vignette is included.

**Comment by AdamSpannbauer** [comment]
Ohhh my b. Was viewing on mobile. Think we can call this closed?

**Comment by bmewing** [comment]
#close
